src/librustdoc/html/static/styles/main.css

```diff
@@ -26,9 +26,6 @@ h1.fqn {
 h2, h3:not(.impl):not(.method):not(.type):not(.tymethod), h4:not(.method):not(.type):not(.tymethod) {
 	border-bottom-color: #DDDDDD;
 }
-.in-band {
-	background-color: white;
-}
```

Comment:
I think the issue is that the target here is the `h3`. The `.in-band` is a child of that `h3`. So its `background` overrides the `background` of its parent.
I'm really not an expert on CSS. Do you know of a way to prevent that behavior, at least in this context?